rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -61,6 +61,7 @@
 #include "namespace_prefix.hpp"
 
 #include "dds/dds.h"
+#include "dds/features.h"
```

Comment:
Oh darn, you're already including dds.h, I missed that. Erik's PR is including features.h within dds.h, so you'll get it transitively here. You can remove this line.

Reply:
Removed.